oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -251,7 +259,14 @@ public APIGatewayProxyResponseEvent handleRequest(
                             .orElseThrow(
                                     () ->
                                             new AuthenticationCallbackException(
-                                                    "Orchestration user session not found"));
+                                                    "Shared session not found in Redis"));
+            OrchSessionItem orchSession =
+                    orchSessionService
+                            .getSession(sessionCookiesIds.getSessionId())
+                            .orElseThrow(
+                                    () ->
+                                            new AuthenticationCallbackException(
```

Comment:
I'm a bit wary of throwing this when the feature is still in development, so may look at feature flagging exceptions thrown when querying Orch session table.